terraform_compliance/steps/steps.py

```diff
@@ -240,20 +240,34 @@ def its_value_must_be_set_by_a_variable(step_obj):
     step_obj.context.stash.property(step_obj.context.search_value).should_match_regex(r'\${var.(.*)}')
 
 
-@then(u'it must not have {proto:ANY} protocol and port {port:ANY} for {cidr:ANY}')
-def it_must_not_have_proto_protocol_and_port_port_for_cidr(step_obj, proto, port, cidr):
+@then(u'it {condition:ANY} have {proto:ANY} protocol and port {port} for {cidr:ANY}')
```

Comment:
Better to define with a separate function like https://github.com/eerkunt/terraform-compliance/blob/55223a65fd5987062f6a9bfae4c71f6c99c19d7c/terraform_compliance/steps/steps.py#L17 to check if it is `only` or `not`.

We want to keep `must` all the time, so you don't need to include `must` in the definitons.